media/js/base/gtm/ie/gtm-snippet-ie-init.es6.js

```diff
@@ -6,65 +6,4 @@
 
 import GTMSnippet from '../gtm-snippet.es6.js';
```

Comment:
**Suggestion/question:** this file is now identical to `gtm-snipper-init.es6.js`, so could both IE and regular browsers now both load the same file? (and we can delete this one?)